AndroidRunner/Plugins/trepn/Trepn.py

```diff
@@ -166,7 +193,8 @@ def aggregate_end(self, data_dir, output_file):
 
     def aggregate_trepn_subject(self, logs_dir):
         def add_row(accum, new):
-            row = {key: value + float(new[key]) for key, value in list(accum.items()) if key not in ['Component', 'count']}
+            row = {key: value + float(new[key]) for key, value in list(accum.items()) if
+                   key not in ['Component', 'count']}
```

Comment:
This and the change below were autoformatted due to the lines being so long.
If prefered, I can undo these changes.